SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -96,7 +96,10 @@ NodeProxy : BusPlug {
 	nodeID { ^group.nodeID }
 
 	parentGroup_ { | node |
-		if(node.isPlaying.not) { "node not playing and registered: % \n".postf(node); ^this };
+		if(node.isPlaying.not) {
+			"% : node not playing and registered: % \n".postf(thisMethod, node);
```

Comment:
+ "node not playing but registered" ?
+ "node neither playing nor registered" ?